src/h3lib/lib/vertex.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file  vertex.h
+ *  @brief Functions for working with cell vertexes.
+ */
+
+#include "vertex.h"
+
+#include "baseCells.h"
+#include "faceijk.h"
+#include "geoCoord.h"
+#include "h3Index.h"
+
+/** @brief Table of direction-to-face mapping for each pentagon
+ *
+ * Note that faces are in directional order, starting at J_AXES_DIGIT.
+ * This table is generated by the generatePentagonDirectionFaces script.
+ */
+static const PentagonDirectionFaces pentagonDirectionFaces[NUM_PENTAGONS] = {
```

Comment:
aside: this table could be used to implement getPentagonIndexes rather than looping through the base cell table.